src/features/unified-share-modal/ContactsField.js

```diff
@@ -50,16 +50,40 @@ class ContactsField extends React.Component<Props, State> {
 
         this.state = {
             contacts: [],
+            numSuggestedShowing: 0,
             pillSelectorInputValue: '',
         };
     }
 
+    addSuggestedContacts = (contacts: Array<Contact>) => {
+        const { suggestedCollaborators = {} } = this.props;
```

Comment:
Maybe we can use `defaultProps` here instead.

Reply:
IMO it's better to use this pattern because it keeps the logic closer to the code and gives more context if that context is needed. For example, you might want to know when the results are empty versus when the prop isn't passed within the component.